womtool/src/main/scala/womtool/wom2wdlom/WomToWdlom.scala

```diff
@@ -221,10 +221,10 @@ object WomToWdlom {
   // WOM has some explicit representations that are implicit in WDL; they are necessary for execution,
   // but do not make sense (or are illegal) in WDL source.
   private def selectWdlomRepresentableNodes(allNodes: Set[GraphNode]): Set[GraphNode] = {
-    val expressions: Set[GraphNode] = allNodes.filterByType[ExposedExpressionNode]
-    val scatters: Set[GraphNode] = allNodes.filterByType[ScatterNode]
-    val calls: Set[GraphNode] = allNodes.filterByType[CallNode]
-    val conditionals: Set[GraphNode] = allNodes.filterByType[ConditionalNode]
+    val expressions = allNodes.filterByType[ExposedExpressionNode]
```

Comment:
examples where the `filterByType` ascriptions are now not only unnecessary but unusable

Reply:
This is my not-exactly-the-same `EnhancedTraversable` => `EnhancedIterableOps` port